### PR TITLE
feat: billing usage resource

### PR DESCRIPTION
**Pull request by LeoAltissimo**
Implements requests to Billing usage. Related to [this endpoint](https://docs.latitude.sh/reference/get-billing-usage)

### CODE REVIEW

**Comment by marcelo-sousa-mh** [comment]
@LeoAltissimo upon testing the dashboard integration, I needed to work on some fixes for this PR.
#1035 will be blocked until we merge this one.

**Comment by LeoAltissimo** [comment]
@marcelo-sousa-mh Thank you for making the changes. The updated version looks good to me.

**Review by marcelo-sousa-mh — APPROVED** [review]

**Pull request merged**
